pkg/os/windows/powershell/powershell_windows.go

```diff
@@ -61,7 +32,7 @@ func IsAdmin() bool {
 
 func Execute(args ...string) (string, string, error) {
 	args = append([]string{"-NoProfile", "-NonInteractive", "-ExecutionPolicy", "RemoteSigned", "-Command"}, args...)
-	cmd := exec.Command(LocatePowerShell(), args...) // #nosec G204
+	cmd := exec.Command("powershell.exe", args...) // #nosec G204
```

Comment:
why not use `exec.LookPath` here like in the `ExecuteAsAdmin` below

Reply:
done!